zetaclient/bitcoin/bitcoin_client.go

```diff
@@ -1217,7 +1225,13 @@ func (ob *BTCChainClient) checkTssOutTxResult(hash *chainhash.Hash, res *btcjson
 	if err != nil {
 		return errors.Wrapf(err, "checkTssOutTxResult: invalid TSS Vin in outTx %s nonce %d", hash, nonce)
 	}
-	err = ob.checkTSSVout(rawResult.Vout, params, nonce)
+
+	// differentiate between normal and banned cctx
+	if clientcommon.IsCctxBanned(cctx) {
+		err = ob.checkTSSVoutCancelled(params, rawResult.Vout)
+	} else {
+		err = ob.checkTSSVout(params, rawResult.Vout)
+	}
 	if err != nil {
 		return errors.Wrapf(err, "checkTssOutTxResult: invalid TSS Vout in outTx %s nonce %d", hash, nonce)
 	}
```

Comment:
I think it would be better to have a different error handling for both cases above and returning a different error message

Reply:
https://github.com/zeta-chain/node/pull/1790/commits/cf72c1bfc036951b792b8d3d7b327fbb7b53cc32